packages/blade/.storybook/react-native/Storybook.tsx

```diff
@@ -1,18 +1,33 @@
 import { AppRegistry } from 'react-native';
 import { getStorybookUI } from '@storybook/react-native';
+import { BladeProvider } from '../../src/components/BladeProvider';
+import { paymentTheme, bankingTheme } from '../../src/tokens/theme';
 import './storybook.requires';
-const Storybook = getStorybookUI({
-  // keeping in comments becuase this is not documented properly in the docs
-  // theme: {
-  //   backgroundColor: 'white',
-  //   headerTextColor: 'black',
-  //   labelColor: 'black',
-  //   borderColor: '#e6e6e6',
-  //   previewBorderColor: '#b3b3b3',
-  //   buttonTextColor: '#999999',
-  //   buttonActiveTextColor: '#444444',
-  // },
-});
+
 import { name as appName } from '../../app.json';
 
-AppRegistry.registerComponent(appName, () => Storybook);
+const App = (): React.ReactElement => {
+  const Storybook = getStorybookUI({
+    // keeping in comments becuase this is not documented properly in the docs
+    // theme: {
+    //   backgroundColor: 'white',
+    //   headerTextColor: 'black',
+    //   labelColor: 'black',
+    //   borderColor: '#e6e6e6',
+    //   previewBorderColor: '#b3b3b3',
+    //   buttonTextColor: '#999999',
+    //   buttonActiveTextColor: '#444444',
+    // },
+  });
+
+  return (
+    <BladeProvider
+      // key={`${context.globals.themeTokens}-${context.globals.colorScheme}`}
+      themeTokens={paymentTheme}
```

Comment:
not putting efforts to add a theme switcher and color mode switcher like we have on web. might add later once the need arises